driver/razermouse_driver.c

```diff
@@ -3359,48 +3446,339 @@ static DEVICE_ATTR(right_matrix_effect_none,        0220, NULL,
 static DEVICE_ATTR(backlight_led_state,            0660, razer_attr_read_backlight_led_state, razer_attr_write_backlight_led_state);
 
 
+#define REP4_DPI_UP  0x20
+#define REP4_DPI_DN  0x21
+#define REP4_TILT_L  0x22
+#define REP4_TILT_R  0x23
+#define REP4_PROFILE 0x50
+#define REP4_SNIPER  0x51
```

Comment:
Where do these names come from?

Reply:
If the names are any indicator: those are for tilting the wheel left and right, the profile switch button, and the detachable "sniper" button.